### ISSUE TITLE
Different results for demo picture 

### ISSUE THREAD

**Issue by SheenaB2** [yizhiwang96/Slice3D#2]
Hi,

Thank you for your excellent work on the Slice3D project! I am trying to run the demo picture of a shoe using the code provided with your pre-trained models. I followed the instructions in the "Setup" and "Testing on Single Image" sections of your README file. However, I obtained different results compared to what is shown in your paper using both the regression-based method and the generation-based method. Specifically, the shape appears different, and for the regression model, there are no holes inside the shoe. I attached screenshot of my results below. 

Could you please let me know if there is anything I might be doing wrong? Thank you!
![regression_based_shoe](https://github.com/yizhiwang96/Slice3D/assets/122312083/31897e9a-90f0-4eb8-9291-2f4059c6eeaa)
![generation_based_shoe](https://github.com/yizhiwang96/Slice3D/assets/122312083/0317b0f0-4d31-43d8-87d5-84ae9a88905d)

**Comment by yizhiwang96** [comment]
Hi @SheenaB2, thank you for your interest in Slice3D! Your produced results are correct.

About your question: why are the results different to the ones in paper?

Ans: I did some data cleaning when releasing the code and ckpts, making the training dataset smaller to the one used in our paper.  Less examples of shoes/boots could be the reason why our method failed in reconstructing the hole in regression-mode. 

I also notice that our regression-based results are generally better than generation-based ones. The main advantage of generation-based slicing is that it can produce multiple results instead of deterministic ones.

About data cleaning: when slice a 3D model, we need the meshes in this 3D model to be [merged](https://github.com/yizhiwang96/Slice3D/blob/d02c0b307c94eb113feb3a5d4f6543cfdaad4dbb/render_slices/blender_script_slices.py#L163). However, Blender could fail in merging meshes for some 3D models from Objaverse, yielding broken structures. I did not include these 3D models in the training dataset.
